openshift/e2e-tests-openshift.sh

```diff
@@ -95,7 +95,7 @@ function install_knative(){
   export IMAGE_webhook=${IMAGE_FORMAT//\$\{component\}/knative-serving-webhook}
   # Kourier is not built in this project.
   # export IMAGE_kourier=${IMAGE_FORMAT//\$\{component\}/kourier}
-  export IMAGE_kourier="quay.io/3scale/kourier:v0.3.11"
+  export IMAGE_kourier="quay.io/3scale/kourier:v0.3.12"
```

Comment:
Shall we use our own kourier versions as built by our fork and used in serverless-operator? Current would be `registry.svc.ci.openshift.org/openshift/knative-v0.13.2:kourier`

Reply:
I thought that we should not hard code `registry.svc.ci.openshift.org` and so avoided to use it. 
 (I remember https://github.com/openshift/knative-serving/pull/401).

But we can use `registry.svc.ci.openshift.org/openshift/knative-v0.13.2:kourier` for this case?

Reply:
Yeah these references are still fine. Only per-built hardcoded references are not at this point.

Reply:
Oh, I see. Then, we should use it and actually we can use the original catalogsource without edit for Kourier. I am running https://github.com/openshift/knative-serving/pull/457 for sure.